packages/core/src/__tests__/ceramic-api.test.ts

```diff
@@ -36,6 +36,9 @@ const generateStringOfSize = (size): string => {
   }
   return random_data.join('')
 }
+// Should  pass on v4 if updated from TileDocument
+const describeIfV3 = process.env.CERAMIC_ENABLE_V4_MODE ? describe.skip : describe
+const skipIfV4ShouldPass = process.env.CERAMIC_ENABLE_V4_MODE ? it.skip : it
```

Comment:
why is this called `skipIfV4ShouldPass` but in other tests we use `testIfV3ShouldPass`?  Can we be consistent so it's easier to grep for these later?